app/components/employer-table/component.js

```diff
@@ -5,8 +5,8 @@ var DATE_FORMAT = 'YYYY-MM-DD';
 
 export default Ember.Component.extend(ShowMore, {
   items: Ember.computed.alias('model.top'),
-  displayDownload: Ember.computed('exportSet', 'session.isLoggedIn', function(){
-    return this.get('session.isLoggedIn') && this.get('exportSet') !== undefined; // prevents download link from appearing if there is no endpoint
+  displayDownload: Ember.computed('exportSet', 'session.isAdmin', function(){
+    return this.get('session.isAdmin') && this.get('exportSet') !== undefined; // prevents download link from appearing if there is no endpoint
```

Comment:
@ryanstgermaine Do you still want any logged in user to see the `Download all records` option?  Or do you want this restricted to admins only?

![image](https://cloud.githubusercontent.com/assets/504505/14928254/99ee7eb6-0e0b-11e6-861a-9934fe065f41.png)